candidates/migrations/0034_candidates_locked_data.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+from django.db import migrations
+
+
+def move_candidates_locked_to_postextraelection(apps, schema_editor):
+    PostExtra = apps.get_model('candidates', 'PostExtra')
+    PostExtraElection = apps.get_model('candidates', 'PostExtraElection')
+
+    locks = PostExtra.objects.filter(candidates_locked=True)
+    for lock in locks:
+        for election in lock.elections.all():
+            new_lock = PostExtraElection.objects.get(
+                postextra=lock,
+                election=election
+            )
+            new_lock.candidates_locked = True
+            new_lock.save()
+
+
+def move_candidates_locked_to_postextra(apps, schema_editor):
+    """
+    Becuase this locks at a post level it means the reverse migration
+    is potentially lossy. In cases where a post has been in more than one
+    election and not all of them are locked that data will be lost.
+    However, it seemed better to be permissive in locking as I think it's
+    easier to spot "Hey, why can't I edit this" that "Hey, I should not be
+    able to edit this". Hopefully. Regardless, if you want to do this
+    you need to set an env variable `ALLOW_LOSSY_REVERSE_MIGRATIONS=1`
+    """
+    if 'ALLOW_LOSSY_REVERSE_MIGRATIONS' not in os.environ:
```

Comment:
So that this matches the comment exactly, I think I'd make the condition `os.environ.get('ALLOW_LOSSY_REVERSE_MIGRATIONS') == '1'`